src/pages/validator/amplifier/verifier-rotations.mdx

```diff
@@ -0,0 +1,53 @@
+# Verifier Rotations
+
+One type of message the network can sign and prove to a destination chain is a message containing an update to the verifier set. To make an update to the verifier set for a chain, the previous verifier set must sign a message communicating the changes.
+
+## Check if a verifier is in an active set
+
+[`query_get_verifier_set()`](https://github.com/axelarnetwork/axelar-amplifier/blob/3a184af756addf2b2e4ed2797933a0e583a6d0f4/contracts/multisig-prover/src/contract.rs#L301) is a function on the multisig prover that gets the active verifier set for a chain. The active verifier set is the same as the signer set unless there is a verifier set update in progress, since the prover requires a few extra steps before active verifier sets can be confirmed as signers.
+
+```bash
+export CHAIN_MULTISIG_PROVER_ADDRESS="axelar1xz4cya4qm2ws6nzperhvc40wdjcq4872fl6d3j2s4cytyx8j80eqenv87g"
+export RPC="http://devnet-verifiers.axelar.dev:26657"
+```
+
+```bash
+axelard q wasm contract-state smart $CHAIN_MULTISIG_PROVER_ADDRESS \
+    '"query_get_verifier_set"'
```

Comment:
this is wrong. Real query is:

`axelard q wasm contract-state smart $CHAIN_MULTISIG_PROVER_ADDRESS     '"current_verifier_set"'  --node $RPC`

Queries don't use gas/wallets.

Reply:
Make sure you are running your commands.